src/util/commandline/command.ts

```diff
@@ -113,6 +115,13 @@ export class Command {
       setQuiet();
     }
 
+    // Check if a new cli version is available
+    const result: Buffer = execSync("npm show appcenter-cli version");
+    const newVersion: string = result.toString("utf-8", 0, result.length - os.EOL.length);
+    if (newVersion !== this.getVersion()) {
+      console.log(`The new version is available: ${newVersion}`);
```

Comment:
Change the wording to:
`A new version of appcenter-cli is available: ${newVersion}`